frontend/src/components/Schedule/Comments.tsx

```diff
@@ -76,46 +116,239 @@ const CommentHeaderWithDropDown = styled.div`
+          variant="outlined"
+          fullWidth
+        />
+        <SubmitCommentButton>
+          <CommentButton
+            onClick={handleCommentButtonClick}
+            disabled={buttonDisabled}
+          >
+            Comment
+          </CommentButton>
+        </SubmitCommentButton>
+      </InputContainer>
+    );
+  };
+
+  const CommentContainer = () => {
+    if (tab !== 0) {
+      return null;
+    }
+
+    return (
+      <>
+        {comments.map((comment: IComment) => (
+          <Comment
+            author={comment.author}
+            comment={comment.comment}
+            createdAt={comment.createdAt}
+            updatedAt={comment.updatedAt}
+          ></Comment>
+        ))}
+      </>
+    );
+  };
+
+  const ChangeLogsContainer = () => {
+    if (tab !== 1) {
+      return null;
+    }
+
+    return (
+      <>
+        {changeLogs.map((changeLog: IChangeLog) => (
+          <ChangeLog
+            userId={changeLog.userId}
+            author={changeLog.author}
+            log={changeLog.log}
+            createdAt={changeLog.createdAt}
+            updatedAt={changeLog.updatedAt}
+          />
+        ))}
+      </>
+    );
+  };
+
+  const handleTabChange = (event: any, newValue: any) => {
+    setTab(newValue);
+  };
+
+  return (
+    <>
+      <CommentsDropdownContainer>
+        <CommentHeaderWithDropDown>
+          <div
+            onClick={() => {
+              setIsExpanded(!isExpanded);
+            }}
+            style={{ marginRight: 4, marginLeft: 2 }}
+          >
+            {isExpanded ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
+          </div>
+          <CommentsHeaderWithTabs>
+            <CommentsHeader>
+              <CommentHeaderText> Comments </CommentHeaderText>
+            </CommentsHeader>
+            {isExpanded && (
+              <TabContainer>
+                <MuiThemeProvider theme={CommentTheme}>
+                  <Tabs
+                    value={tab}
+                    onChange={handleTabChange}
+                    variant="fullWidth"
+                    indicatorColor="secondary"
+                    textColor="secondary"
+                    aria-label="icon label tabs example"
+                  >
+                    <Tab label="COMMENTS" />
+                    <Tab label="CHANGE HISTORY" />
+                  </Tabs>
+                </MuiThemeProvider>
+              </TabContainer>
+            )}
+          </CommentsHeaderWithTabs>
+        </CommentHeaderWithDropDown>
+        <ContentContainer>
+          {isExpanded && (
+            <div>
+              {loading ? (
+                <LinearLoading />
+              ) : (
+                <>
+                  <CommentContainer />
+                  <ChangeLogsContainer />
+                  <ContentEnd ref={contentEndRef}></ContentEnd>
+                </>
+              )}
+            </div>
+          )}
+        </ContentContainer>
+        <CommentInput />
```

Comment:
It feels a little weird/unintuitive that you can add comments when on the change log tab since it feels like it's indicating you can comment on the change log. I'd say move it to just comments unless you have some other reason for keeping it for both

Reply:
Yea that makes sense, I wanted to make commenting at any point super easy. Maybe we take users back to the comment tab if they comment?